leaf-ui/js/backendComm.js

```diff
@@ -2,19 +2,21 @@
"1110","1110"]}]},{"intentionElements":[{"id":"0","status":["0010","0010"]},{"id":"1","status":["0000","0000"]}]},{"intentionElements":[{"id":"0","status":["0010","0010"]},{"id":"1","status":["0011","0011"]}]},{"intentionElements":[{"id":"0","status":["0010","0010"]},{"id":"1","status":["0010","0010"]}]},{"intentionElements":[{"id":"0","status":["0010","0010"]},{"id":"1","status":["0100","0100"]}]},{"intentionElements":[{"id":"0","status":["0010","0010"]},{"id":"1","status":["0110","0110"]}]},{"intentionElements":[{"id":"0","status":["0010","0010"]},{"id":"1","status":["0111","0111"]}]},{"intentionElements":[{"id":"0","status":["0011","0011"]},{"id":"1","status":["1110","1110"]}]},{"intentionElements":[{"id":"0","status":["0011","0011"]},{"id":"1","status":["1100","1100"]}]},{"intentionElements":[{"id":"0","status":["0011","0011"]},{"id":"1","status":["0011","0011"]}]},{"intentionElements":[{"id":"0","status":["0011","0011"]},{"id":"1","status":["0010","0010"]}]},{"intentionElements":[{"id":"0","status":["0011","0011"]},{"id":"1","status":["0000","0000"]}]},{"intentionElements":[{"id":"0","status":["0011","0011"]},{"id":"1","status":["0100","0100"]}]},{"intentionElements":[{"id":"0","status":["0011","0011"]},{"id":"1","status":["0110","0110"]}]},{"intentionElements":[{"id":"0","status":["0011","0011"]},{"id":"1","status":["0111","0111"]}]},{"intentionElements":[{"id":"0","status":["1110","1110"]},{"id":"1","status":["0100","0100"]}]},{"intentionElements":[{"id":"0","status":["1110","1110"]},{"id":"1","status":["0111","0111"]}]},{"intentionElements":[{"id":"0","status":["1110","1110"]},{"id":"1","status":["0110","0110"]}]},{"intentionElements":[{"id":"0","status":["1110","1110"]},{"id":"1","status":["0000","0000"]}]},{"intentionElements":[{"id":"0","status":["1110","1110"]},{"id":"1","status":["0011","0011"]}]},{"intentionElements":[{"id":"0","status":["1110","1110"]},{"id":"1","status":["0010","0010"]}]},{"intentionElements":[{"id":"0","status":["1110","1110"]},{"id":"1","status":["1110","1110"]}]},{"intentionElements":[{"id":"0","status":["1110","1110"]},{"id":"1","status":["1100","1100"]}]},{"intentionElements":[{"id":"0","status":["1100","1100"]},{"id":"1","status":["1100","1100"]}]},{"intentionElements":[{"id":"0","status":["1100","1100"]},{"id":"1","status":["1110","1110"]}]},{"intentionElements":[{"id":"0","status":["1100","1100"]},{"id":"1","status":["0110","0110"]}]},{"intentionElements":[{"id":"0","status":["1100","1100"]},{"id":"1","status":["0111","0111"]}]},{"intentionElements":[{"id":"0","status":["1100","1100"]},{"id":"1","status":["0100","0100"]}]},{"intentionElements":[{"id":"0","status":["1100","1100"]},{"id":"1","status":["0010","0010"]}]},{"intentionElements":[{"id":"0","status":["1100","1100"]},{"id":"1","status":["0011","0011"]}]},{"intentionElements":[{"id":"0","status":["1100","1100"]},{"id":"1","status":["0000","0000"]}]}],"finalAssignedEpoch":[],"finalValueTimePoints":[],"relativeTimePoints":5,"absoluteTimePoints":[1,3]};
```

Comment:
Why are you committing hard coded values??

Reply:
This is just an example if you want to execute in localhost.
But I can remove, no problem.